### ISSUE TITLE
Remove stage-0 Babel preset

### ISSUE THREAD

**Issue by eliperelman** [mozilla-neutrino/neutrino-dev#11]
I'd like to remove `babel-preset-stage-0` from the React preset. The only real reason I had it included was it was needed in the past for Object rest spread. I think we should swap stage-0 out for `babel-plugin-transform-object-rest-spread`. The proposal is current at stage-3.

Looking at the stage presets for Babel, at the moment there is nothing really useful other than spread. I'll address this in the plugin work.

**Comment by eliperelman** [comment]
Fixed in #10.

**Comment by atoiv** [comment]
For anyone wondering how to use the removed stage-0 babel-preset with react preset, the following [custom config](https://neutrino.js.org/customization/advanced.html) does seem to do the trick:

```javascript
// my-custom-config.js
const merge = require('deepmerge');

module.exports = neutrino => {
  neutrino.config
    .module
      .rule('compile')
        .loader('babel', ({ options }) => {
          return {
            options: merge(options, {
              presets: [
                require.resolve('babel-preset-stage-0'),
              ],
            })
          };
        });
}
```

For me, the missing plugin was [stage-2](https://github.com/tc39/proposal-class-public-fields) [`babel-plugin-transform-class-properties`](https://babeljs.io/docs/plugins/transform-class-properties/), for class properties. Which in react's context is handy for auto binding event handlers, and using propTypes as static class properties. But there's probably other features in stage-0 I'd want to use as well.
